src/main/java/igrad/ui/CommandBox.java

```diff
@@ -22,9 +24,14 @@
 
     private final CommandExecutor commandExecutor;
 
+    private static String animalGuideIcon;
+
     @FXML
     private TextField commandTextField;
 
+    @FXML
+    private ImageView animalGuide;
```

Comment:
this line's prob redundant, there's no such GUI component in the `CommandBox.fxml` file, I'm removing it. Actually the ImageView to display the Uwu in the command box is in `ResultDisplay.fxml` file instead.